cmd/pull.go

```diff
@@ -38,16 +40,14 @@ func NewPullCommand() *cobra.Command {
 }
 
 func pullImage(cmd *cobra.Command, args []string) error {
-	podmanSocket, err := cmd.Flags().GetString("podman-socket")
+	cOpts, err := cmdutil.GetCommonOpts(cmd)
 	if err != nil {
 		return err
 	}
 
 	ctx := context.Background()
-	hnd, err := podman.NewHandle(ctx, podmanSocket)
-	if err != nil {
-		return err
-	}
+
+	hnd, err := podman.NewHandle(ctx, cOpts.PodmanSocket, cmdutil.NewLogger(cOpts.Verbose, 0))
```

Comment:
I think you removed error handling by mistake

Reply:
Indeed. Thanks.